src/django/api/views/v1/moderation_events.py

```diff
@@ -81,26 +87,11 @@ def retrieve(self, _,  pk=None):
 
     @handle_errors_decorator
     def patch(self, request, pk=None):
-        if not (request.user.is_superuser or request.user.is_staff):
-            raise PermissionDenied(
-                detail="Only the Moderator can perform this action."
-            )
+        self.__validate_user_permissions(request)
 
-        if not ModerationIdValidator.is_valid_uuid(pk):
-            return handle_path_error(
-                field="moderation_id",
-                message="Invalid UUID format.",
-                status_code=status.HTTP_400_BAD_REQUEST
-            )
+        self.__validate_uuid(pk)
```

Comment:
@mazursasha1990 Did you test it after changes?
Also probably we need to discuss and/or update API spec as message will used build-in ViewSet error.